src/routes/api/users.js

```diff
@@ -1,12 +1,19 @@
 import { Router } from 'express';
 import { SendVerificationEmail, handleInvalidEmail, handleEmptyEmailBody } from '../../middlewares/mail';
 import emailverification from '../../controllers/emailController';
+import UserController from '../../controllers/userController';
+import Helper from '../../middlewares/index'
+import { validateProfileData } from '../../validation/userValidation'
 
 const router = Router();
 
+router.get('/users/:id', UserController.getUserProfile);
+router.patch('/users/:id', validateProfileData, Helper.verifyToken, UserController.updateUserProfile);
```

Comment:
I will suggest you make the route /users/:userId/profile

Reply:
Okay